src/perception/camera_object_detection/config/traffic_signs_config.yaml

```diff
@@ -0,0 +1,9 @@
+traffic_signs_node:
+  ros__parameters:
+    camera_topic: /camera/left/image_color
+    publish_vis_topic: /traffic_signs_viz
+    publish_detection_topic: /traffic_signs
+    model_path: /perception_models/traffic_signs_v1.pt
+    crop_mode: CenterCrop
+    image_size: 1024
```

Comment:
@Gongsta I was mislead by this parameter because it actually doesn't mean the size of the input image from the camera topic, its actually the size in which the image is reshaped before going into the object detector.

How camera-agnostic is Yolov8?

Reply:
The reason for `CenterCrop` and specifying `image_size` is so that the predictions can be more accurate. The `image_size` was initially, by default, set to `640` which removes some detail from the image which reduces prediction accuracy since objects being detected like traffic lights and traffic signs are really small and get blurred out by the resizing (done by Letterbox too). `CenterCrop` crops the image so we don't lose too much detail when resizing.

I'm not sure by what you mean by camera-agnostic. The dataset the Yolov8 model is trained on is a collection of images from left camera, right camera, and center camera (that has fisheye lens). We're planning on removing images from center camera since it might affect how signs are detected in general. According to Steven, the left camera has the most high resolution pictures.

Reply:
nw about this @Miniapple8888 , i was asking cuz I feel like the ros input image size should be specified in the config rather than determined on the fly. But I guess the current way is fine too as it reduces the number params we have to always reset and whatnot